devtools_m2/cypress/integration/CustomerCartInteractions/steps.js

```diff
@@ -14,6 +14,8 @@ When('I create an account', function() {
     cy.get('input[name="password_confirmation"]').type('blahblah123!!!')
     cy.contains('Create an Account').click()
   })
+  cy.get('.message-success > div').contains('Thank you for registering with')
+  cy.visit('/lib/web/blank.html')
```

Comment:
☝️ Not really part of the issue, but I found and edge case where waiting for the registration process to complete helped the stability the test platform.